org.raceintospace.Raceintospace.yml

```diff
@@ -0,0 +1,89 @@
+# vim: sts=4:
+app-id: org.raceintospace.Raceintospace
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: raceintospace
+#rename-icon: raceintospace
+rename-desktop-file: raceintospace.desktop
+rename-appdata-file: raceintospace.appdata.xml
+modules:
+    - name: sdl
+      buildsystem: autotools
+      sources:
+        - type: archive
+          url: http://libsdl.org/release/SDL-1.2.15.tar.gz
+          sha256: d6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00
+        - type: patch
+          path: SDL-XData32.patch
+      cleanup:
+        - SDL*.3
+        - '/lib/*.a'
+        - '/include'
+    - name: jsoncpp
+      buildsystem: cmake
+      sources:
+        - type: git
+          url: https://github.com/open-source-parsers/jsoncpp.git
+          tag: 1.9.3
+      cleanup:
+        - '/lib/*.a'
+        - '/include'
+    - name: physfs
+      buildsystem: cmake
+      sources:
+        - type: archive
+          url: https://icculus.org/physfs/downloads/physfs-3.0.2.tar.bz2
+          sha256: 304df76206d633df5360e738b138c94e82ccf086e50ba84f456d3f8432f9f863
+    - name: protobuf
+      buildsystem: autotools
+      sources:
+        - type: git
+          url: https://github.com/protocolbuffers/protobuf.git
+          tag: v3.13.0
+      cleanup:
+        - '/lib/libproto*.a'
+        - '/include'
+    - name: protobuf-c
+      buildsystem: autotools
+      sources:
+        - type: git
+          url: https://github.com/protobuf-c/protobuf-c.git
+          tag: v1.3.3
+    - name: boost
```

Comment:
Boost build should be limited to specific modules the game needs.

Reply:
I used --with-headers only. I don't know how to choose any more specific. Is there shared knowledge base for common modules? Any project with boost usage?

Reply:
Oh, it seems install target needs it also. Thanks, fixed.